src/app/cli/src/coda_main.ml

```diff
@@ -51,15 +51,7 @@ end
 module Ledger_hash = struct
   include Ledger_hash
 
-  let of_digest = Ledger_hash.of_digest
-
-  let merge = Ledger_hash.merge
-
-  let to_bytes = Ledger_hash.to_bytes
-
-  let of_digest = Ledger_hash.of_digest
-
-  let merge = Ledger_hash.merge
+  let of_digest, merge, to_bytes = Ledger_hash.(of_digest, merge, to_bytes)
```

Comment:
oh, very nice shorthand. I didn't realize module items take patterns on the left hand side. of course they do though.

Reply:
wow that's really cool

Reply:
this changes everything

Reply:
I'm glad to be of service. :-)